client/tests/test_client.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from client import models
+from client.models import Client
+from django.core.exceptions import ValidationError
+
+
+USER_NAME_TEST = 'liorsil'
+FIRST_NAME_TEST = 'lior'
+LAST_NAME_TEST = 'silber'
+PASSWORD_TEST = '1234qwer'
+USER_TYPE_TEST = 'client'
+AREA_TEST = 'yuvalim'
+
+
+@pytest.fixture
+def client0():
+    return models.Client(user_name=USER_NAME_TEST,
+                         first_name=FIRST_NAME_TEST,
+                         last_name=LAST_NAME_TEST,
+                         password=PASSWORD_TEST,
+                         user_type=USER_TYPE_TEST,
+                         area=AREA_TEST)
+
+
+@pytest.fixture
+def saved_client0(client0):
+    client0.save()
+    return client0
+
+
+class TestClientModel:
+    @pytest.mark.django_db()
+    def test_save_client(self, client0):
+        """Test - checks if client is saved in DB.
+        Returns:
+            None.
+        """
+        Client.save_client(client0)
+        assert client0 in models.Client.objects.all()
```

Comment:
How do you know it wasn't there before?

Reply:
I can add in line 37: assert client0 not in models.Client.objects.all()

so it would prove that client0 not in the table before. 
this is what you mean right?

Reply:
yes